queries.sql

```diff
@@ -1,3 +1,17 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+
+SELECT * FROM animals WHERE date_of_birth > '2016-01-01' AND date_of_birth < '2019-01-01';
```

Comment:
- [ ] Please fix this query and update the screenshot, because you found the birth dates older than '2016-1-1' and younger than '2019-1-1'. However, you are asked to find animals which are 'born between 2016 and 2019.' Which also includes the first day of 2016 and the last day of 2019.